docs/meshstack.logging.md

```diff
@@ -120,6 +120,72 @@ It should be considered for the future to also log the userId, to correlate log
 
 All log file entries always start with an UTC datetime. This is followed by the log-level (DEBUG, INFO, WARN, ERROR, ...).
 
+### Log in Meshfed
+
+We've configured our Spring Boot Applications (meshfed, kraken and identityconnector) to support log level changing during runetime. To expose this functionality we're using actuator configuration
+to expose `/actuator/loggers` endpoint to request all available components with there current log level.
+
+For example:
+
+```JSON
+curl --location --request GET 'http://localhost:8080/actuator/loggers' \
+--header 'Authorization: Basic <username-password>'
+
+{
+    "levels": [
+        "OFF",
+        "ERROR",
+        "WARN",
+        "INFO",
+        "DEBUG",
+        "TRACE"
+    ],
+    "loggers": {
+        "ROOT": {
+            "configuredLevel": "INFO",
+            "effectiveLevel": "INFO"
+        },
+        "com": {
+            "configuredLevel": null,
+            "effectiveLevel": "INFO"
+        },
+        "com.rabbitmq": {
```

Comment:
d: Better provide an example for "io.meshcloud". And perhaps one for rabbitmq and one for http logging

Reply:
Fixed.